frontend/src/environments/environment.brc4env.ts

```diff
@@ -10,7 +16,13 @@ export const environment: DataDiscoveryEnvironment = {
     secondLogoUrl: '',
     links: [{ label: 'brc4env', url: 'https://www.brc4env.fr/' }]
   },
-  resourceModule: RareModule,
+  documentComponent: RareDocumentComponent,
+  documentListComponent: GenericDocumentListComponent,
+  ontologyAggregationComponent: GenericOntologyAggregationComponent,
+  headerComponent: RareHeaderComponent,
+  footerComponent: GenericFooterComponent,
+  basketComponent: GenericBasketComponent,
```

Comment:
This looks wrong. The `basket.enabled` property is true, and the RareModule, that was previously used (`resourceModule: RareModule`), had `BasketComponent` in its declarations and exports.

Reply:
You're right, good catch